src/routes/event-team.tsx

```diff
@@ -195,11 +184,36 @@ const EventTeam = ({ eventKey, teamNum }: Props) => {
       back={`/events/${eventKey}`}
       class={eventTeamStyle}
     >
-      {nextMatch && (
+      {upcomingMatches && upcomingMatches.length > 0 ? (
         <>
-          <h2 class={sectionStyle}>Next Match</h2>
-          <MatchDetailsCard match={nextMatch} eventKey={eventKey} link />
+          <Heading level={2}>Upcoming Matches</Heading>
+          <div
+            class={css`
+              display: grid;
+              grid-template-columns: 20rem;
+              justify-self: stretch;
+            `}
+          >
+            {upcomingMatches.map((match) => (
+              <MatchDetailsCard
+                key={match.key}
+                match={match}
+                eventKey={eventKey}
+                link
+              />
+            ))}
+          </div>
         </>
+      ) : (
+        <p
+          class={css`
+            color: gray;
+            text-align: center;
+            margin: 1rem 0 0.8rem;
+          `}
+        >
+          No Upcoming Matches
+        </p>
       )}
```

Comment:
@varCepheid I am wondering how helpful it is to have this "no upcoming matches" text:

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/539a425b-c9b6-48f8-88b5-a8fb068be9d0)

What do you think about just leaving that part empty if there aren't any coming up?